src/Login/action_types.js

```diff
@@ -0,0 +1,6 @@
+export default {
+    USERNAME: 'LOGIN_USERNAME',
```

Comment:
I personally prefer to have action_types fields match up with its string value. Like `LOGIN_USERNAME`: `LOGIN_USERNAME`